syntax = "proto3";

option java_package = "org.etri.ado";
option optimize_for = SPEED;

package org.etri.ado;

message DoubleValue {
  double value = 1;
}

message FloatValue {
  float value = 1;
}

message Int64Value {
  int64 value = 1;
}

message UInt64Value {
  uint64 value = 1;
}

message Int32Value {
  int32 value = 1;
}

message UInt32Value {
  uint32 value = 1;
}

message BoolValue {
  bool value = 1;
}

message StringValue {
  string value = 1;
}

message BytesValue {
  bytes value = 1;
}

message Empty {

}
